apps/src/sites/studio/pages/levels/_curriculum_reference.js

```diff
@@ -1,6 +1,16 @@
-window.curriculumReference = {
-  onload: onload
-};
+import $ from 'jquery';
+import { registerGetResult } from '@cdo/apps/code-studio/levels/codeStudioLevels';
+import { postMilestoneForPageLoad, onContinue } from '@cdo/apps/code-studio/levels/postOnLoad';
+
+$(document).ready(() => {
```

Comment:
I've always wondered: Is there a reason we don't use the shorthand `$(fn)` for `$(document).ready(fn)`? ([docs](https://learn.jquery.com/using-jquery-core/document-ready/))